flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/sql/SqlITCase.scala

```diff
@@ -696,6 +696,181 @@ class SqlITCase extends StreamingWithStateTestBase {
       "6,8,Hello world,51,9,5,9,1")
     assertEquals(expected.sorted, StreamITCase.testResults.sorted)
   }
+  
+     //////////////////////////////////////////////////////
+   // START TESTING BOUNDED PROC TIME ROW AGGREGATION
+   //////////////////////////////////////////////////////
+  
+
+  @Test
+  def testSumMinAggregatation2(): Unit = {
```

Comment:
@fhueske suggested the 4 test, and 4 I implemented. It is the 5th time I implement the tests, we can leave it like that. :-)